Gemfile.lock

```diff
@@ -1,146 +1,155 @@
+GIT
+  remote: https://github.com/envygeeks/jekyll-assets
+  revision: 056d2c88719ef3b1f90967a606dd1441581dd832
+  specs:
+    jekyll-assets (4.0.0.alpha)
```

Comment:
alpha?

Reply:
We have to use the latest possible version to get it to work with the Jekyll 4 updates.